guest-relations.md

```diff
@@ -1,6 +1,7 @@
 # Guest Relations
 
-This committee is responsible for ensuring the safety and to ensure a pleasant experience to our foreign and local guests (especially the speakers).
+- Past Definition: This committee is responsible for ensuring the safety and to ensure a pleasant experience to our foreign and local guests (especially the speakers).
+- Current Definition: Guide guests and answer random queries, Help Desk, Usher, Crowd Control
```

Comment:
Change to: 

- This committee is responsible for ensuring the safety and to ensure a pleasant experience to our foreign and local guests (especially the speakers). 

- In addition, this committee will be guiding guests online and offline. This is typically done by answering random queries, be on the help desk, usher, and crowd control.